app/views/questions/_answer.html.erb

```diff
@@ -36,8 +36,8 @@
     <% end %>
     <hr />
     <div style="padding: 0px 20px 10px 20px;">
-    <% if current_user && current_user.uid == answer.author.uid || answer.accepted %>
-      <a <% if current_user && current_user.uid == answer.author.uid %> href="/answers/accept/<%= answer.id %>" <% end %> data-remote="true" id="answer-<%= answer.id %>-accept" class="answer-accept btn btn-sm <% if answer.accepted %>btn-success<% else %>btn-default<% end%>"><% if answer.accepted %>Accepted<% else %>Accept this answer<% end %></a>
+    <% if current_user && current_user.uid == answer.node.uid || answer.accepted %>
+      <a <% if current_user && current_user.uid == answer.node.uid %> href="/answers/accept/<%= answer.id %>" <% end %> data-remote="true" id="answer-<%= answer.id %>-accept" class="answer-accept btn btn-sm <% if answer.accepted %>btn-success<% else %>btn-default<% end%>"><% if answer.accepted %>Accepted<% else %><% if current_user && current_user.uid == answer.node.uid %>Accept this answer<% end %><% end %></a>
```

Comment:
For readability (and because the first is not actually a working button, even if it uses similar styling), could you break out the `Accepted` and `Accept this answer` messages into two separate lines, and two separate `<a>` tags?